primary-maternal-care/1B.md

```diff
@@ -1,210 +1,878 @@
 ---
 book: Primary Maternal Care
-title: 1B. Examination of the abdomen in pregnancy
+title: 1. Antenatal care
```

Comment:
@arthurattwell The content from Maternal Care replaces older material from Primary Maternal Care.